pkg/apis/velero/v2alpha1/data_upload_types.go

```diff
@@ -90,7 +90,6 @@ const (
 	DataUploadPhaseAccepted   DataUploadPhase = "Accepted"
 	DataUploadPhasePrepared   DataUploadPhase = "Prepared"
 	DataUploadPhaseInProgress DataUploadPhase = "InProgress"
-	DataUploadPhaseCanceling  DataUploadPhase = "Canceling"
```

Comment:
> Canceling: the data upload is being canceled
Canceled: the data upload has been canceled

[From design](https://github.com/vmware-tanzu/velero/blob/main/design/volume-snapshot-data-movement/volume-snapshot-data-movement.md)

Given that there could be more data movers in the future using Data Upload CRs. Do we wanna consider a case where a data mover can take longer to cancel something? We need a way to distinguish between actually cancelled, and an "in-progress towards cancellation".

We should keep this const for future data movers IMO.

Reply:
@kaovilai @shubham-pampattiwar If the DU/DD controllers never reference this phase, then wouldn't some external mover setting it break the workflow?

Reply:
@sseago won't the external datamover controllers work on the DU CRs ?

Reply:
Hmm. I'm not 100% clear on exactly how that will work, but in any case, shouldn't there be a common workflow/state model? Open to suggestion here, though -- lets see what other maintainers say.